bin/cml/repo/prepare.test.js

```diff
@@ -0,0 +1,30 @@
+const { exec } = require('../../../src/utils');
```

Comment:
delete this file?

Reply:
https://github.com/iterative/cml/pull/1073#discussion_r957920154